zeppelin-server/src/test/java/org/apache/zeppelin/cluster/ClusterEventTest.java

```diff
@@ -66,9 +66,7 @@
 import java.util.Map;
 import java.util.Set;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertThat;
+import static org.junit.Assert.*;
```

Comment:
I would prefer a list of imports and no wildcard.

Reply:
checkstyle plugin should catchup this, at least in main code....